metaflow/includefile.py

```diff
@@ -44,6 +49,35 @@ def size(self):
         return self._size
 
 
+class MultipleFiles(object):
+    def __init__(self, base_name, logger, is_text, encoding):
+        self._base_name = base_name
+        self._logger = logger
+        self._is_text = is_text
+        self._encoding = encoding
+        self._files = {}
+
+    def add_file(self, path):
+        name = os.path.basename(path)
+        # Sanitize the name to make it possible to use as a variable name
```

Comment:
@romain-intel what's the idea behind this logic? Why would you want to mangle the file name?

Reply:
To give a simple name and an unambiguous one. I could change the logic to remove the random bit though (and just through an error if I can't create a simple name). The idea of prefetching the name of the parameter is to allow, for example, to import different similarly structured data sets. It also reduces the change of colliding with reserved names.

Reply:
I still don't quite follow. Could you give an example of a `path` and the various transformation that `name` goes through?